subprojects/platform-base/src/integTest/groovy/org/gradle/language/base/LanguageTypeSampleIntegrationTest.groovy

```diff
@@ -27,15 +28,14 @@ class LanguageTypeSampleIntegrationTest extends AbstractIntegrationSpec {
     @Rule
     Sample languageTypeSample = new Sample(temporaryFolder, "customModel/languageType")
 
-    def setup() {
-        executer.usingInitScript(RepoScriptBlockUtil.createMirrorInitScript())
-    }
-
     def "shows custom language sourcesets in component"() {
         given:
         sample languageTypeSample
+        RepoScriptBlockUtil.replaceExternalRepos(testDirectory)
```

Comment:
Because this sample contains `buildSrc`: https://github.com/gradle/gradle/blob/a1bdf232d9afea4ff4964c1c70db8c298d11a7cb/subprojects/docs/src/samples/customModel/languageType/buildSrc/build.gradle#L20

Seems like init script can't configure buildSrc project.

What do you think we should do for the cases which can't use init script? Should we leave them as they are or rewrite their build scripts?